CODIGO/TileEngine_Chars.bas

```diff
@@ -325,6 +325,53 @@ Char_Move_by_Head_Err:
     
 End Sub
 
+Public Sub TranslateCharacterToPos(ByVal charindex As Integer, ByVal NewX As Integer, ByVal NewY As Integer, ByVal TranslationTime As Long)
+On Error GoTo TranslateCharacterToPos_Err
+    Dim TileX, TileY As Integer
+    Dim DiffX, DiffY As Integer
+100 With charlist(charindex)
+102     TileX = .Pos.x
+104     TileY = .Pos.y
+106     If Not InMapBounds(TileX, TileY) Then Exit Sub
+        Debug.Assert MapData(TileX, TileY).charindex = charindex
+        If MapData(TileX, TileY).charindex <> charindex Then
+        Dim ftx, fty As Integer
+        For ftx = 1 To 99
```

Comment:
Quite expensive these nested for loops. Does `TranslateCharacterToPos` get called a lot? 

From what I see you only use the loop to do a debug.print, it would be better if we guard this with `#if DEBUGGING`

Reply:
ops this was debug coded that I forgot to remove